BEIMA.DB.Schemas/device-type-schema.txt

```diff
@@ -12,4 +12,22 @@
         "customFieldUid2": String,
         "customFieldUid3": String
     }
+}
+
+What the front end will send to the endpoint when creating a device type:
+
+{
+    "_id": ObjectId,
+    "name": String,
+    "description": String,
+    "notes": String,
+    "lastModified": {
+        "date": Date,
+        "username": String
+    },
+    "fields": [
+        "customFieldUid1",
```

Comment:
These should be the field names rather than the Uids I think.

Reply:
You're right, fixed [7504052](https://github.com/cs481-ekh/s22-beima/pull/206/commits/7504052a95b6b77b0306c8930067fb1e841c0591)